src/compiler/diagnosticMessages.json

```diff
@@ -1687,6 +1691,10 @@
         "category": "Error",
         "code": 2528
     },
+    "Duplicate identifier '{0}'. Compiler reserves name '{1}' in top level scope of a module containing async functions.": {

```

Comment:
These also?

* [line 818](https://github.com/Microsoft/TypeScript/blob/master/src/compiler/diagnosticMessages.json#L818)
* [line 1190](https://github.com/Microsoft/TypeScript/blob/master/src/compiler/diagnosticMessages.json#L1190)
* [line 1198](https://github.com/Microsoft/TypeScript/blob/master/src/compiler/diagnosticMessages.json#L1198)
* [line 1206](https://github.com/Microsoft/TypeScript/blob/master/src/compiler/diagnosticMessages.json#L1206)

If you want to change the various messages we use for these cases, I'd rather we do that in a different PR.